gitlint-core/gitlint/tests/contrib/rules/test_authors_commit.py

```diff
@@ -0,0 +1,88 @@
+from collections import namedtuple
+from unittest.mock import Mock, patch
+
+import pytest
```

Comment:
Easiest is to not import pytest at all, that way we don't rely on pytest specific methods.